src/Authentication/Controllers/SystemRegisterController.cs

```diff
@@ -68,7 +73,7 @@ public async Task<ActionResult<RegisteredSystemDTO>> GetRegisteredSystemDto(stri
     {
         RegisteredSystem registeredSystem = await _systemRegisterService.GetRegisteredSystemInfo(systemId, cancellationToken);
         
-        if (registeredSystem == null)
+        if (registeredSystem == null || !registeredSystem.IsVisible)
```

Comment:
I think this will break the bff and frontend? For vendorrequest and change request, in GET, the request only contains the systemId. The bff uses this service to load the system (for displaying orgno, vendorname, systemname). 

An alternative solution for this is for backend to include the system object in GET vendorrequest and change request